addons/l10n_sa_edi/tests/test_edi_zatca.py

```diff
@@ -325,3 +325,37 @@ def testInvoiceWithRetention(self):
             move_data=move_data,
             freeze_time_at=datetime(2022, 9, 5, 8, 20, 2, tzinfo=timezone('Etc/GMT-3'))
         )
+
+    def testCompanyOnSimplifiedInvoiceQR(self):
+        move_data = {
+            'name': 'INV/2025/00012',
+            'invoice_date': '2025-07-05',
+            'invoice_date_due': '2025-07-12',
+            'partner_id': self.partner_sa_simplified,
+            'invoice_line_ids': [{
+                'product_id': self.product_a.id,
+                'price_unit': self.product_a.standard_price,
+                'quantity': 1,
+                'tax_ids': self.tax_15.ids,
+            }],
+        }
+
+        # Fetch company name from xml
+        invoice = self._create_invoice(**move_data)
+        invoice.company_id = self.sa_branch
```

Comment:
Change made: Remove the quantity as it is one by default, put the company in the move directly

```suggestion
        # Fetch company name from xml
        invoice = self._create_invoice({
            'name': 'INV/2025/00012',
            'company_id': self.sa_branch,
            'invoice_date': '2025-07-05',
            'invoice_date_due': '2025-07-12',
            'partner_id': self.partner_sa_simplified,
            'invoice_line_ids': [{
                'product_id': self.product_a.id,
                'price_unit': self.product_a.standard_price,
                'tax_ids': self.tax_15.ids,
            }],
        })
```